api/controllers/tasks.py

```diff
@@ -120,6 +125,23 @@ def process_proposal(credentials, tpid):
             last_updated=db.sql.func.now(),
         )  # Annotation config is sentiment example.
 
+        emailReceiver = tp_creator_email
+        cc = "no-reply@dynabench.org"
+        subject = "Dynabench Task Approval Mail"
+
+        try:
+            mail.send(
+                server="",
```

Comment:
i'm assuming this is debug code along with your changes to `config.py.example` above, this should probably be the `bottle.default_app().config["mail"]` that is set in `server.py`